dali/python/nvidia/dali/ops.py

```diff
@@ -152,16 +197,7 @@ def _docstring_prefix_from_inputs(op_name):
     # __call__ docstring
     ret += "\nOperator call to be used in graph definition.\n"
     # Args section
-    ret += """
-Args
-----
-"""
-    for i in range(schema.MaxNumInput()):
-        optional = i >= schema.MinNumInput()
-        input_type_str = schema.GetInputType(i) + _supported_layouts_str(schema.GetSupportedLayouts(i))
-        ret += _numpydoc_formatter(schema.GetInputName(i), input_type_str, schema.GetInputDox(i), optional)
-        ret += "\n"
-    ret += "\n"
+    ret += _get_inputs_doc(schema)
```

Comment:
Should we mention that the same set of `kwargs` as for `__init__` is accepted here?